aea/cli_gui/__init__.py

```diff
@@ -104,10 +104,7 @@ def _call_subprocess(*args, timeout=None, **kwargs):
             )
```

Comment:
changes in this file were due to an error when waiting for the termination in a test (see below)